apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/ResourceNames.cs

```diff
@@ -337,7 +337,7 @@ public partial class Session
         /// </summary>
         public SessionName SessionName
         {
-            get { return string.IsNullOrEmpty(Name) ? null : Google.Cloud.Spanner.V1.SessionName.Parse(Name); }
+            get { return string.IsNullOrEmpty(Name) ? null : SessionName.Parse(Name); }
```

Comment:
Revert? (Presumably this is a R# auto-simplification. But basically we don't want to change generated code.)